diff --git a/node_modules/react-native-callkeep/android/src/main/java/io/wazo/callkeep/RNCallKeepModule.java b/node_modules/react-native-callkeep/android/src/main/java/io/wazo/callkeep/RNCallKeepModule.java
index 025480a..8f43a21 100644
--- a/node_modules/react-native-callkeep/android/src/main/java/io/wazo/callkeep/RNCallKeepModule.java
+++ b/node_modules/react-native-callkeep/android/src/main/java/io/wazo/callkeep/RNCallKeepModule.java
@@ -434,7 +434,6 @@ public class RNCallKeepModule extends ReactContextBaseJavaModule implements Life
         this.hasListeners = false;
     }
 
-    @ReactMethod
     public void displayIncomingCall(String uuid, String number, String callerName) {
         this.displayIncomingCall(uuid, number, callerName, false, null);
     }
@@ -444,7 +443,6 @@ public class RNCallKeepModule extends ReactContextBaseJavaModule implements Life
         this.displayIncomingCall(uuid, number, callerName, false, null);
     }
 
-    @ReactMethod
     public void startCall(String uuid, String handle, String contactIdentifier) {
         startCall(uuid, handle, contactIdentifier, false);
     }
@@ -533,7 +531,6 @@ public class RNCallKeepModule extends ReactContextBaseJavaModule implements Life
         reportEndCallWithUUID(uuid, reason);
     }
 
-    @ReactMethod
     public void reportEndCallWithUUID(String uuid, int reason) {
         if (!isConnectionServiceAvailable() || !hasPhoneAccount()) {
             return;
@@ -628,7 +625,6 @@ public class RNCallKeepModule extends ReactContextBaseJavaModule implements Life
         this.endCall(uuid);
     }
 
-    @ReactMethod
     public void endCall(String uuid) {
         Log.d(TAG, "[RNCallKeepModule] endCall called");
         if (!isConnectionServiceAvailable() || !hasPhoneAccount()) {